playgrounds/vite-cloudflare-environment/svelte.config.js

```diff
@@ -0,0 +1,16 @@
+import { vitePreprocess } from '@sveltejs/vite-plugin-svelte';
+import adapter from '@sveltejs/adapter-auto';
+import { cloudflare } from '@flarelabs-net/vite-environment-provider-cloudflare';
+
+/** @type {import('@sveltejs/kit').Config} */
+const config = {
+	kit: {
+		adapter: adapter(),
+		environments: {
+			ssr: cloudflare()
```

Comment:
I think this should be specified by using the cloudflare adapter instead and I would remove this config option

Reply:
Agreed. I'll work on integrating with the existing adapter API instead.